internal/kafka/internal/presenters/object_reference.go

```diff
@@ -41,6 +43,8 @@ func objectKind(i interface{}) string {
 		return KindError
 	case api.ServiceAccount, *api.ServiceAccount:
 		return KindServiceAccount
+	case api.Cluster, *api.Cluster:
+		return KindCluster
```

Comment:
great catch!

I wondering if this was ever returned by the registration endpoint, do you know?

Reply:
I will check. if it wasn't, I will add it

Reply:
I think it not. It was not returned. I am thinking it might be good to return them now. https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1446#discussion_r1046825981